kafka-connector/README.md

```diff
@@ -169,4 +185,51 @@ from a Pubsub message into a SourceRecord with a relevant Schema.
     *   In these cases, to carry forward the structure of data stored in
         attributes, we recommend using a converter that can represent a struct
         schema type in a useful way, e.g. JsonConverter.
+        
+        
+#### Pub/Sub Lite Connector
+
+Pub/Sub Lite's messages have the following structure:
+
+```java
+class Message {
+  ByteString key;
+  ByteString data;
+  ListMultimap<String, ByteString> attributes;
+  Optional<Timestamp> eventTime;
+}
+```
  
+This maps quite closely to the SinkRecord class, except for serialization. The
+table below shows how each field in SinkRecord will be mapped to the underlying
+message:
+
+| SinkRecord | Message |
+|---|---|
+| key{Schema} | key |
+| value{Schema} | data |
+| headers | attributes |
+| topic | attributes["x-goog-pubsublite-source-kafka-topic"] |
+| kafkaPartition | attributes["x-goog-pubsublite-source-kafka-partition"] |
+| kafkaOffset | attributes["x-goog-pubsublite-source-kafka-offset"] |
+| timestamp | eventTime |
+| timestampType | attributes["x-goog-pubsublite-source-kafka-event-time-type"] |
+
+When a key, value or header value with a schema is encoded as a ByteString, the
+following logic will be used:
+
+- null schemas are treated as Schema.STRING_SCHEMA
+- Top level BYTES payloads are unmodified.
+- Top level STRING payloads are encoded using copyFromUtf8.
+- Top level Integral payloads are converted using
+copyFromUtf8(Long.toString(x.longValue()))
+- Top level Floating point payloads are converted using
+copyFromUtf8(Double.toString(x.doubleValue()))
+- All other payloads are encoded into a protobuf Value, then converted to a ByteString.
```

Comment:
We may ultimately need to make this configurable. For people who don't use proto at all, they probably aren't going to want to have to convert to using it. In particular, generic protocol buffers like this can be a little hard to use, I think. Granted, the vast majority of people should probably  just be using a BytesConverter that treats the payload as BYTES, which leaves the data unmodified. Maybe don't need to do anything about it now, but just something to keep in mind.